### ISSUE TITLE
adding a form for submitting sites to be displayed in the site

### ISSUE THREAD

**Issue by centurion6x** [yashikabhargava/WebX#130]
## Description
Users will be able to suggest websites/resources to be included to the site admins via a new suggestions form. It will take in the link, a description, the type of resource and the user's information. 

## Screenshots
NA

## Do you want to work on this issue?
Yes

## How are you planning to resolve on this issue?
A new form and a page will be created using the existing pages and styles as a blueprint. 

## Are you raising this issue under hacktoberfest? 
Yes, I'm raising this issue under hacktoberfest.

**Comment by yashikabhargava** [comment]
@centurion6x That is a great suggestion, but since this is a static website, I do not feel the need to add this form currently but may be in the future.

For now, we can do one thing, there can be a section that says "have some suggestions or want to add new resources" and that section can redirect to this GitHub repo. Something like the mdn web doc website.
![des](https://user-images.githubusercontent.com/76060661/197820968-cdcefc00-ca0b-4b59-b620-e4961c6e7561.PNG)
here is the link to the website - https://developer.mozilla.org/en-US/docs/Learn/JavaScript

Do tell me if you like the suggestion or not and if would you like to work on it.

**Comment by centurion6x** [comment]
Yes, that's a great idea. I'd love to work on it. Shall I open a new issue?

**Comment by yashikabhargava** [comment]
@centurion6x You can edit this one only.

**Comment by centurion6x** [comment]
Ah okie
